blog/_posts/2016-07-14-ola-from-l1ghtsab3r.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Introducing Team l1ghtsab3r India
+layout: post
+created_at: Thur Jul 14 2016
+permalink: blog/introducing-team-l1ghtsab3r-india
+current: blog
+author: Team l1ghtsab3r
+categories:
+- student-posts
+---
+
+
+### About Us <img src="../img/blog/2016/l1ghtsab3r-saber-purple.png" style="height: 25px; width: 60px">
```

Comment:
please remove the `..` at the beginning of the image paths, the link works without that :)